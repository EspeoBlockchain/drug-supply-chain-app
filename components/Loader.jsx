import React from 'react';
import {
  View,
  ActivityIndicator,
  StyleSheet,
} from 'react-native';
import colors from '../constants/Colors';
import layout from '../constants/Layout';

const styles = StyleSheet.create({
  container: {
    justifyContent: 'center',
    alignItems: 'center',
    width: layout.window.width,
    height: 50,
    backgroundColor: colors.registerLight,
    alignSelf: 'center',
    flex: 0,
  },
  activityIndicator: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});

const loader = () => (
  <View style={styles.container}>
    <ActivityIndicator
      animating
      color={colors.registerDark}
      size="large"
      style={styles.activityIndicator}
    />
  </View>
);

export default loader;
