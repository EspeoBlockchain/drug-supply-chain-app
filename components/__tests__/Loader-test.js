import 'react-native';
import React from 'react';
import Loader from '../Loader';
import renderer from 'react-test-renderer';

it('renders corectly', () => {
    const tree = renderer.create(<Loader />).toJSON();

    expect(tree).toMatchSnapshot();
});