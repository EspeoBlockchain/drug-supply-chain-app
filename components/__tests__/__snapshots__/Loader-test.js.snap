// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corectly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#f7a58a",
      "flex": 0,
      "height": 50,
      "justifyContent": "center",
      "width": 750,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#d9572c"
    hidesWhenStopped={true}
    size="large"
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  />
</View>
`;
